test/integration/integration-test.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
+var _ = require('lodash');
+var assert = require('assert');
+var supertest = require('supertest');
+var wallet = require('./wallet');
+var fixtures = require('../fixtures');
+
+var PREFIX = '/v1';
+var TIMEOUT = 20000;   // how long before each test case times out
+var INTERVAL = 2000;   // how long to wait between checks for validated ledger
+var API = supertest('http://localhost:5990');
+
+
```

Comment:
Intentional, the style guide currently allows up to two consecutive blank lines.